.github/workflows/build-images.yml

```diff
@@ -44,6 +45,31 @@ jobs:
           docker run $LOGIN_SERVER/$IMAGE:$SHA_TAG /bin/bash \
             -c "conda run --no-capture-output -n condaenv pytest tests/test_packages.py"
 
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+          cache: "pip"
+
+      - name: Install requirements
+        if: ${{ matrix.proc == 'cpu' }}
+        run: pip install -r requirements.txt
```

Comment:
Why do we need to setup Python?

Reply:
Generating the quickstart submission relies on numpy & pandas to generate random descriptors